components/kyma-environment-broker/internal/auditlog/config.go

```diff
@@ -1,9 +1,48 @@
 package auditlog
 
+import (
+	"errors"
+	"fmt"
+	"text/template"
+
+	"github.com/Masterminds/semver"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/auditlog/templates"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/cls"
+)
+
 type Config struct {
 	URL           string `envconfig:"APP_AUDITLOG_URL"`
 	User          string `envconfig:"APP_AUDITLOG_USER"`
 	Password      string `envconfig:"APP_AUDITLOG_PASSWORD"`
 	Tenant        string `envconfig:"APP_AUDITLOG_TENANT"`
 	EnableSeqHttp bool   `envconfig:"APP_AUDITLOG_ENABLE_SEQ_HTTP"`
 }
+
+type Overrides struct {
+	Host         string
+	Port         string
+	Path         string
+	HttpPlugin   string
+	ClsOverrides *cls.ClsOverrideParams
+	Config       Config
+}
+
+func GetExtraConfTemplate(KymaVersion string) (*template.Template, error) {
+	c, err := semver.NewConstraint("< 1.20.x")
+	if err != nil {
+		return nil, errors.New("unable to parse constraint for kyma version %s to set correct fluent bit plugin")
+	}
+
+	var version *semver.Version
```

Comment:
This creation is unnecessary. We can create and assign `version` at the same time in the next line.

Reply:
done